src/raft.rs

```diff
@@ -861,6 +869,7 @@ impl<T: Storage> Raft<T> {
 
         let last_index = self.raft_log.last_index();
         let committed = self.raft_log.committed;
+        self.uncommitted_size = self.compute_uncommitted_size();
```

Comment:
`reset` is called by `become_follower`, `become_candidate` and `become_leader`, yet `uncommitted_size` is only useful for the leader, instead of `reset`, I think `become_leader` will be a better place for this line of code.

Reply:
Done, PTAL:)